dspace/config/dspace.cfg

```diff
@@ -846,6 +846,26 @@ event.consumer.ldnmessage.filters = Item+Install
 event.consumer.submissionconfig.class = org.dspace.submit.consumer.SubmissionConfigConsumer
 event.consumer.submissionconfig.filters = Collection+Modify_Metadata
 
+# The CurationTaskConsumer starts a curation task as a process in DSpace.
+# The curation task runs as its own process and therefore is not depending
+# on the session of the current user.
+# If shows up in the list of processes in the admin process panel in the web ui.
+# It runs on the subject of the DSpace Event that was fired and can be used
+# on any dspace object type that is used as subject of an event (like items,
+# collections and communities).
+# To configure it, give it a name, reference the class and configure the
+# name of the curation task you want it to schedule and start:
+#
+# event.consumer.noopCurator.class = org.dspace.curate.CurationTaskConsumer
+# event.consumer.noopCurator.filters = Item+Install|Item+Modify_Metadata
```

Comment:
`event.consumer.noopCurator.filters = Item+Install|Item+Modify_Metadata` 
This configuration is not working for me, I think you should use `Item+Install|Modify_Metadata`.
But in the end I wouldn't use that configuration in real env because the curation would be scheduled many times for the same item.
I think curation should be run only once for an Item or re-run if something changes / goes wrong.
I would leave the Install config, but not the Modify_Metadata one.